lib/vdsm/storage/lvmconf.py

```diff
@@ -184,3 +189,27 @@ def save(self):
     def close(self):
         log.debug("Closing LVM configuration %s", self.path)
         self.aug.close()
+
+
+def configured_value(option):
+    """
+    Return value configured for the option, taken into account all config
+    files or default configurations. Option should be single LVM config
+    option, i.e. when querying e.g. whole configuration section, which
+    returns multiple configuration values, an exception will be thrown.
+    When option is not found, command fails and exception is thrown directly
+    from commands module.
+    """
+    cmd = [constants.EXT_LVM, "lvmconfig", "--typeconfig", "full", option]
+    out = commands.run(cmd).decode("utf-8").strip()
+
+    if "=" not in out or "\n" in out:
+        raise se.UnexpectedLvmConfigOutput(option, out)
+    else:
+        value = out.split("=")[1]
+        # String value.
+        if value.startswith('"'):
+            return value[1:len(value) - 1]
```

Comment:
Maybe:

    if value[0] == '"' and value[-1] == '"]:
        value = value[1:-1]